jsapp/js/components/submissions/table.es6

```diff
@@ -461,6 +465,10 @@ export class DataTable extends React.Component {
         case ADDITIONAL_SUBMISSION_PROPS._submitted_by:
             index = 'z92';
             break;
+        // set index for `background-audio` to the front of every column with `_`
+        case 'background-audio':
```

Comment:
I added `background-audio` in Form Builder, made few submissions, but I don't see this column in the table. Should it appear?

Reply:
Not sure if you did this but the question will only have submissions if you use ODK/Kobo collect--but I guess the column should be there even if there are no submissions for the question type? This is true for other types (maybe this is what you meant :P)

Reply:
Figured out why the column won't show if there's not `background-audio` data, it's because `getSurveyFlatPaths` doesn't include meta be default. Since we're treating `background-audio` like a meta question, it doesn't actually get sent to us with the submission unless it contains data. What do you think we should do? Or should we leave it as is?

Reply:
Oh right. Let's leave it as is then. It is consistent with other meta questions :+1: